.buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py

```diff
@@ -704,6 +704,8 @@ def tox_factors_for_folder(tests_folder_name: str) -> List[str]:
             # airflow
             AvailablePythonVersion.V3_12,
         ],
+        timeout_in_minutes=40,
+        queue=BuildkiteQueue.DOCKER,
```

Comment:
I think running on the bigger box sounds promising. I don't think we should extend the timeout - it looks like the agent itself is dying and it's not actually hitting a timeout. So perhaps an OOM or segfault or something.